Formula/flif.rb

```diff
@@ -0,0 +1,75 @@
+class Flif < Formula
+  desc "Free Loseless Image Format"
+  homepage "http://flif.info/"
+  head "https://github.com/FLIF-hub/FLIF.git"
+
+  stable do
+    # patch 'install -s', see https://github.com/FLIF-hub/FLIF/issues/369
+    patch :DATA
```

Comment:
@MikeMcQuaid I asked in my first comment about this patch -- is it common issue that `install -s` fails?
Anyways, I've did it: FLIF-hub/FLIF#382.